druid-shell/src/platform/gtk/screen.rs

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020 The druid Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! GTK Monitors and Screen information.
+
+use crate::screen::Monitor;
+use crate::kurbo::Size;
+
+pub fn get_display_size() -> Size {
```

Comment:
Maybe this should be `pub(crate)`? I think it's inaccessible outside `druid-shell` anyway, because we don't export the `platform` module, but `pub(crate)` would be clearer anyway.

Reply:
I re-exported screen in druid so it would be called by Screen::get_display_size();

Reply:
Ah yes. I missunderstood you, they should be pub(crate). Will update this